packages/fxa-content-server/server/lib/routes/post-metrics.js

```diff
@@ -142,7 +141,6 @@ const BODY_SCHEMA = {
     })
     .required(),
   service: STRING_TYPE.regex(SERVICE_PATTERN).required(),
-  settingsVersion: SETTINGS_VERSION.optional(),
```

Comment:
This shouldn't reject requests with that have this property right? Which might happen if were are switching between versions in the deploy.

Reply:
im not 100% sure we allow unspecified fields. we could spread this over two trains but i don't think it's that terrible to lose a few of these while the train is mid-deploy 🤷‍♂️